onnxruntime/test/optimizer/initializer_test.cc

```diff
@@ -203,24 +225,24 @@ void TestInitializerDataField() {
       tensor_proto.add_##type##_data(data[idx]);                 \
     }                                                            \
                                                                  \
-    Initializer init(tensor_proto, Path());                      \
+    const Initializer init(tensor_proto, Path());                \
                                                                  \
     for (size_t idx = 0; idx < data.size(); idx++) {             \
       EXPECT_EQ(data[idx], init.data<type>()[idx]);              \
     }                                                            \
   }
 
 typedef int64_t int64;
-TestInitializerDataFieldSpecialized(float)
-TestInitializerDataFieldSpecialized(double)
-TestInitializerDataFieldSpecialized(int64)
+TestInitializerDataFieldSpecialized(float);
+TestInitializerDataFieldSpecialized(double);
+TestInitializerDataFieldSpecialized(int64);
 
 TEST(OptimizerInitializerTest, DataField) {
   TestInitializerDataField<int8_t>();
   TestInitializerDataField<uint8_t>();
   TestInitializerDataField<int32_t>();
   TestInitializerDataField<int64_t>();
-  TestInitializerDataField<uint16_t>();
+  TestInitializerDataField<MLFloat16>();
```

Comment:
Previously uint16_t is used for both FP16 and BF16, since now we use the exact type, can we also add test for BFloat16?

Reply:
I am not seeing it necessary since it is the same code.